src/main/java/org/junit/runners/ParentRunner.java

```diff
@@ -285,11 +312,13 @@ private void runChildren(final RunNotifier notifier) {
         final RunnerScheduler currentScheduler = scheduler;
         try {
             for (final T each : getFilteredChildren()) {
-                currentScheduler.schedule(new Runnable() {
-                    public void run() {
-                        ParentRunner.this.runChild(each, notifier);
-                    }
-                });
+                if (!isIgnored(each)) {
```

Comment:
Please add a comment explaining why you are ignoring methods here (like "if the statement is ignored, a notification would have been (or would be) done via the code in someMethodOverThere")